definitions/json/AuthorisationCaseField.json

```diff
@@ -14585,6 +14585,339 @@
     "UserRole": "et-acas-api",
     "CRUD": "R"
   },
+
+  {
+    "CaseTypeId": "ET_EnglandWales",
+    "CaseFieldID": "updateReferCaseTo",
+    "UserRole": "caseworker-employment-legalrep-solicitor",
```

Comment:
would remove access to these fields for the solicitor role. I know they don't have access to the event but shouldn't have access to the fields as well

Reply:
removed